action.yaml

```diff
@@ -78,14 +84,22 @@ runs:
 
         echo Scanning Docker container ${CONTAINER_IMAGE} with Mondoo Client...
         mondoo scan --detect-cicd docker://${CONTAINER_IMAGE} --config mondoo.json --score-threshold ${{ inputs.score_threshold }} --output ${{ inputs.output_format }} ${{ inputs.extra_args }}
-      if: inputs.scan_type == 'docker_image_from_dockerfile' || inputs.scan_type == 'docker_image'
-    - name: Scan configs
+      if: inputs.scan_type == ^docker_image$|^docker_image_from_dockerfile$ && inputs.scan_type != ^k8s$|^terraform$
+    # Only run on k8s manifests, where ```--path``` is required for manifests.
+    - name: Scan k8s manifest
       shell: bash
       run: |
         echo Scanning ${{ inputs.path }} with Mondoo Client...
         mondoo scan --detect-cicd ${{ inputs.scan_type }} --path ${{ inputs.path }} --config mondoo.json --score-threshold ${{ inputs.score_threshold }} --output ${{ inputs.output_format }} ${{ inputs.extra_args }}
-      if: inputs.scan_type != 'docker_image_from_dockerfile' && inputs.scan_type != 'docker_image'
-          
+      if: inputs.scan_type == 'k8s' && inputs.scan_type != ^docker_image$|^docker_image_from_dockerfile$|^terraform$
```

Comment:
could this be easier ? `inputs.scan_type == 'k8s'`

Reply:
Resolving as having discussion in other thread!